_topic_maps/_topic_map.yml

```diff
@@ -2560,6 +2560,8 @@ Topics:
     File: logging-5-7-release-notes
 - Name: Support
   File: cluster-logging-support
+- Name: Getting started with Logging
```

Comment:
IMHO this should go after the About section, i.e. first what is logging, second, how do I use it?

Reply:
I think you're right, good call!